mods/ts/rules/gdi-vehicles.yaml

```diff
@@ -68,6 +68,8 @@ HVR:
 		Speed: 99
 		Locomotor: hover
 		TerrainOrientationAdjustmentMargin: -1
+		TurnsWhileMoving: true
+		TurnSpeed: 20
```

Comment:
The default turnspeed for vehicles in TS is 20, so this is redundant. Though looking at videos of TS, it feels like they turn a bit faster in vanilla

Reply:
make it to 30

Reply:
30 is definitely much. I wouldn't be worried about it in this PR

Reply:
24